.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+services:
+  - docker
+
+before_install:
+  - curl -L -o docker-build https://raw.githubusercontent.com/mate-desktop/mate-dev-scripts/travis/travis/docker-build
```

Comment:
After merge that PR, the url here should be changed to https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/docker-build.

Reply:
done.